src/client/entity-editor/relationship-editor/relationship-editor.tsx

```diff
@@ -267,11 +277,31 @@ class RelationshipModal
 		});
 	};
 
+	handleNumberAttributeChange = ({target}) => {
+		const value = target.value === '' ? null : target.value;
+		const attributeNumber = {
+			attributeType: 2,
+			value: {textValue: value}
+		};
+		const attributePosition = {
+			attributeType: 1,
+			value: {textValue: null}
+		};
```

Comment:
Is it required to set the position attribute in this method? Does that mean changing the number attribute resets the position attribute to null?
From the name, I would expect it to only change the number attribute.

What will happen once we implement other relationship attributes like date? Will we have to set each attribute not concerned by the handler to null (i.e. in handleNumberAttributeChange set position and date to null, in handleDateAttributeChange set number and position to null, etc.) ? If so we should try to avoid that situation.

Reply:
> What will happen once we implement other relationship attributes like date? Will we have to set each attribute not concerned by the handler to null (i.e. in handleNumberAttributeChange set position and date to null, in handleDateAttributeChange set number and position to null, etc.) ? If so we should try to avoid that situation.

My bad, actually I should give it a different name,  maybe **handleOrdinalAttributeChange.**(This will update both number and position)

And for clarity by no means **handleDateAttributeChange** will have to set number or position inside  it to null.

> Is it required to set the position attribute in this method? Does that mean changing the number attribute resets the position attribute to null?

Hmm, this is a problem. I need to think about it.